core/trino-main/src/main/java/io/trino/execution/SqlTaskManager.java

```diff
@@ -156,15 +153,13 @@ public SqlTaskManager(
 
         this.localMemoryManager = requireNonNull(localMemoryManager, "localMemoryManager is null");
         DataSize maxQueryMemoryPerNode = nodeMemoryConfig.getMaxQueryMemoryPerNode();
-        DataSize maxQueryTotalMemoryPerNode = nodeMemoryConfig.getMaxQueryTotalMemoryPerNode();
-        queryMaxMemoryPerTask = nodeMemoryConfig.getMaxQueryTotalMemoryPerTask();
+        queryMaxMemoryPerTask = nodeMemoryConfig.getMaxQueryMemoryPerTask();
```

Comment:
preexistent - can you still rename this to maxQueryMaxMemoryPerTask

Reply:
you mean to swap "query" and "max"? let's follow up